SSA/Projects/FullyHomomorphicEncryption/Basic.lean

```diff
@@ -34,19 +34,6 @@ variable (q t : Nat) [ hqgt1 : Fact (q > 1)] (n : Nat)
 -- see: https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/Way.20to.20recover.20computability.3F/near/322382109
 -- and :https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/Groebner.20bases
 
--- Question: Can we make something like d := 2^n work as a macro?
```

Comment:
Why is this comment removed? This is still an interesting/valid question

Reply:
Right. I will add it back.

Reply:
https://github.com/opencompl/ssa/pull/192